build-and-test.sh

```diff
@@ -11,12 +11,12 @@ GIT_TAG="${GIT_TAG:-$GIT_BRANCH}"
 PLATFORM="${PLATFORM:-linux/amd64}"
 
 # generate and build dockerfile
-docker buildx build --load --platform=${PLATFORM} --tag image_pipenv --file test/Dockerfile test/
+docker buildx build --load --platform="${PLATFORM}" --tag image_pipenv --file test/Dockerfile test/
 docker run --rm \
     --volume /var/run/docker.sock:/var/run/docker.sock \
     --volume "$(pwd):/$(pwd)" \
     --workdir "$(pwd)" \
     --env PIPENV_CACHE_DIR="$(pwd)/.pipenv" \
     --env GIT_TAG="${GIT_TAG}" \
     --env PY_COLORS=1 \
-    ${enter} image_pipenv
+    "${enter}" image_pipenv
```

Comment:
It's this line.

Output from running `build-and-test.sh` on this branch:

```
PY_COLORS=1  image_pipenv
```

Output from another branch

```
PY_COLORS=1 image_pipenv
```

Note the additional space that has been introduced